docs/package.json

```diff
@@ -1,23 +1,10 @@
 {
-  "name": "lit-element-docs",
-  "version": "1.0.0",
-  "description": "lit-element documentation",
-  "main": "index.js",
+  "private": true,
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "polymer build && jekyll build",
```

Comment:
any problem with adding `npm install` to the build command?

```
"build": "npm install && polymer build && jekyll build",
```

Reply:
I don't think that's a good idea - `npm i` is only run once, then `npm run build` after local doc modification. Also, running `npm i` from a package.json script seems like a bad idea (you could imagine what happens if you put `npm i` in `postinstall`...)

Reply:
I get it :) Thanks!